magento/pwa-studio
# [PWA-713] Update Venia's filters

## Description

Adjust `FILTERS` modal styles to match the new design

## Related Issue
https://jira.corp.magento.com/browse/PWA-713

## Acceptance 
- Verify filters modal matches new design
- Ensure there is no broken functionality with the styling change

### Verification Stakeholders
@dpatil-magento 
@schensley 
@jimbo 

### Specification



### Verification Steps



1. Go to a category page
2. Click on `Filter` button
3. View the new design for the Filter drawer
4. Verify all inputs are positioned and relevant labels are readable
5. Verify current filter pills display as expected
6. Verify `Clear all` button clears current filter list
7. Verify filters apply as expected in the product results


## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
